src/main/java/com/tlswe/awsmock/common/util/PropertiesUtils.java

```diff
@@ -117,4 +117,16 @@ public static String getProperty(final String propertyName) {
         return ret;
     }
 
+
+    /**
+     * Get int type of property value by name.
+     *
+     * @param propertyName
+     *            name of the property to get
+     * @return int type of value
+     */
+    public static int getIntFromProperty(final String propertyName) {
+        return Integer.parseInt(properties.getProperty(propertyName));
```

Comment:
I think you may catch the NumberFormatException here and return 0 by default. Add the code for exception handling before merging the pull request or correct me here - thanks!